frontend/pages/home/index.vue

```diff
@@ -91,4 +97,11 @@ const event: Event = {
   date: new Date(),
   supporters: 10,
 };
+
+const user: User = {
+  name: "John A. Tester",
+  location: "Testville, TN",
```

Comment:
This here is exactly what I need to be reading late at night when I'm doing code reviews and need a laugh 😅